content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -192,6 +198,12 @@ characters.
 
 A Job also needs a [`.spec` section](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
 
+### Job Labels
+
+In 1.27, Job labels will have `batch.kubernetes.io/` prefix for `job-name` and `controller-uid`.
+In releases prior to 1.27, Jobs used unprefixed labels ie `job-name` and `controller-uid`.  
+Both labels will be present for now but the legacy labels `job-name` and `controller-uid` are considered to be deprecated.
```

Comment:
I'd remove this.

We don't describe old behavior. And the new labels are already documented as well-known labels.

Reply:
Done.